core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycleSpec.scala

```diff
@@ -362,6 +365,8 @@ class BpmBillingProjectLifecycleSpec extends AnyFlatSpec {
         .landingZoneId(landingZoneId)
         .jobReport(new JobReport().id(landingZoneJobId.toString))
     )
+    when(workspaceManagerDAO.deleteLandingZone(landingZoneId, testContext))
+      .thenReturn(new DeleteAzureLandingZoneResult().jobReport(new JobReport().id("fake-id")))
```

Comment:
This test point runs through the cleanup code, so needs to return a `DeleteAzureLandingZoneResult` to avoid NullPointerExceptions.